### DISCUSSION TITLE
Hunyuan errors in Automatic1111 ComfyUI

### DISCUSSION THREAD

**Discussion by NikolaSage** [Comfy-Org/ComfyUI#6299] [General]
I keep getting this error: 
DownloadAndLoadHyVideoTextEncoder
No such file or directory:  "C:\\Auto1111\\ComfyUI\\ComfyUI_windows_portable\\ComfyUI\\models\\LLM\\llava-llama-3-8b-text-encoder-tokenizer\\model-00001-of-00004.safetensors".
 Any ideas for a fix?

**Comment by ltdrdata** [comment]
`DownloadAndLoadHyVideoTextEncoder` node belongs to ComfyUI-HunyuanVideoWrapper.
Please move this issue to the repo.
https://github.com/kijai/ComfyUI-HunyuanVideoWrapper

**Comment by NikolaSage** [comment]
Thank you for looking at this!
I have tried to re-install/re-run and it always hangs up here and does not finish.
